docs/docs/draft/documentation/README.md

```diff
@@ -1312,12 +1312,76 @@ curl -XDELETE http://localhost:5681/meshes/mesh-1/traffic-logs/tl-1
 The [`kumactl`](/kumactl) CLI under the hood makes HTTP requests to this API.
 :::
 
+## Security
+
+Kuma helps you secure your current infrastructure with mTLS. The following sections cover details of how it works.
+
+### Certificates
+
+Kuma uses a built-in CA (Certificate Authority) to issue certificates for dataplanes. The root certificate is unique for each mesh
+in the system. On Kubernetes, certificates are stored as a [Kubernetes Secret](https://kubernetes.io/docs/concepts/configuration/secret/).
+On Universal, we leverage the same storage (Postgres) that is used for storing policies.
+
+Certificates generated by Kuma are X.509 certificates that are [SPIFFE](https://github.com/spiffe/spiffe/blob/master/standards/X509-SVID.md) compliant. The SAN of certificate is set to `spiffe://<mesh name>/<service name>`
+
+Currently, Kuma only supports self-signed certificates (`builtin`). In the future, we plan to add support for third-party Certificate Authorities.
```

Comment:
Which certificates? Root CA certificates and/or Dataplane certificate ?

Reply:
Root CA certs, right? Because dataplane certs are signed by root ca cert.